offix/src/main/java/org/aerogear/offix/callbacks/OffixConflictCallback.kt

```diff
@@ -0,0 +1,55 @@
+package org.aerogear.offix.callbacks
+
+import android.util.Log
+import com.apollographql.apollo.api.Error
+import com.apollographql.apollo.exception.ApolloException
+import com.apollographql.apollo.interceptor.ApolloInterceptor
+import org.aerogear.offix.ConflictResolutionHandler
+import org.aerogear.offix.conflictedMutationClass
+import org.aerogear.offix.interfaces.ConflictResolutionInterface
+
+/*Callback class which handles conflicts.
+ */
```

Comment:
I like that callback is file on it is own. this makes easy to see/replicate and link from docs.